test_elasticsearch_dsl/test_document.py

```diff
@@ -1,11 +1,23 @@
 import pickle
+from mock import Mock
 from hashlib import md5
 from datetime import datetime
 
+from elasticsearch_dsl.connections import connections
 from elasticsearch_dsl import document, field, Mapping
 from elasticsearch_dsl.exceptions import ValidationException
 
-from pytest import raises
+from pytest import raises, fixture
+
+@fixture(scope='module')
+def setup_mock_connection(request):
```

Comment:
You should be able to just use existing `mock_client` - https://github.com/elastic/elasticsearch-dsl-py/blob/master/test_elasticsearch_dsl/conftest.py#L40